sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/binaryfile/BinaryFileFormatSuite.scala

```diff
@@ -39,6 +41,12 @@ class BinaryFileFormatSuite extends QueryTest with SharedSQLContext with SQLTest
 
   private var fs: FileSystem = _
 
+  private var file1Status: FileStatus = _
```

Comment:
If we only unit test `buildReader`, we don't need to share those objects across tests.